packages/discord.js/src/util/DataResolver.js

```diff
@@ -55,10 +60,11 @@ function resolveGuildTemplateCode(data) {
 
 /**
  * Data that can be resolved to give a Buffer. This can be:
- * * A Buffer
- * * The path to a local file
- * * A URL <warn>When provided a URL, discord.js will fetch the URL internally in order to create a Buffer.
- * This can pose a security risk when the URL has not been sanitized</warn>
+ * - A Buffer
+ * - The path to a local file
+ * - A URL <warn>When provided a URL, discord.js will fetch the URL internally in order to create a Buffer.
+ * - This can pose a security risk when the URL has not been sanitized</warn>
```

Comment:
Not sure if this will be parsed correctly, but this wasn't supposed to be a separate list item

```suggestion
 * - A URL <warn>When provided a URL, discord.js will fetch the URL internally in order to create a Buffer.
 * This can pose a security risk when the URL has not been sanitized</warn>
```